app/scripts/modules/oracle/serverGroup/details/rollback/rollbackServerGroup.controller.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+let angular = require('angular');
+
+import {ACCOUNT_SERVICE} from 'core/account/account.service';
+import {SERVER_GROUP_WRITER} from 'core/serverGroup/serverGroupWriter.service';
+import {TASK_MONITOR_BUILDER} from 'core/task/monitor/taskMonitor.builder';
+
+module.exports = angular.module('spinnaker.oraclebmcs.serverGroup.details.rollback.controller', [
+      ACCOUNT_SERVICE,
+      require('core/application/modal/platformHealthOverride.directive.js'),
+      require('core/task/modal/reason.directive.js'),
+      SERVER_GROUP_WRITER,
+      TASK_MONITOR_BUILDER,
+      require('../../../common/footer.directive.js'),
```

Comment:
This could be `oraclebmcs/common/footer.directive.js`, which would be helpful because there are a lot of folders named `common` in Deck.